xml/System.Windows.Forms/MessageBoxButtons.xml

```diff
@@ -1,206 +1,207 @@
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_Winforms/Classic MessageBox Example/CS/source.cs" id="Snippet1":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_Winforms/Classic MessageBox Example/VB/source.vb" id="Snippet1":::  
+  
+ ]]></format>
+    </remarks>
+    <altmember cref="T:System.Windows.Forms.MessageBoxDefaultButton" />
+  </Docs>
+  <Members>
+    <Member MemberName="AbortRetryIgnore">
+      <MemberSignature Language="C#" Value="AbortRetryIgnore" />
+      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Windows.Forms.MessageBoxButtons AbortRetryIgnore = int32(2)" />
+      <MemberSignature Language="DocId" Value="F:System.Windows.Forms.MessageBoxButtons.AbortRetryIgnore" />
+      <MemberSignature Language="VB.NET" Value="AbortRetryIgnore" />
+      <MemberSignature Language="F#" Value="AbortRetryIgnore = 2" Usage="System.Windows.Forms.MessageBoxButtons.AbortRetryIgnore" />
+      <MemberSignature Language="C++ CLI" Value="AbortRetryIgnore" />
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>1.0.5000.0</AssemblyVersion>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.MessageBoxButtons</ReturnType>
+      </ReturnValue>
+      <MemberValue>2</MemberValue>
+      <Docs>
+        <summary>The message box contains Abort, Retry, and Ignore buttons.</summary>
+      </Docs>
+    </Member>
+    <Member MemberName="CancelTryContinue">
+      <MemberSignature Language="C#" Value="CancelTryContinue" />
+      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.Windows.Forms.MessageBoxButtons CancelTryContinue = int32(6)" />
+      <MemberSignature Language="DocId" Value="F:System.Windows.Forms.MessageBoxButtons.CancelTryContinue" />
+      <MemberSignature Language="VB.NET" Value="CancelTryContinue" />
+      <MemberSignature Language="F#" Value="CancelTryContinue = 6" Usage="System.Windows.Forms.MessageBoxButtons.CancelTryContinue" />
+      <MemberSignature Language="C++ CLI" Value="CancelTryContinue" />
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.MessageBoxButtons</ReturnType>
+      </ReturnValue>
+      <MemberValue>6</MemberValue>
+      <Docs>
+        <summary>Specifies that the message box contains Cancel, Try Again, and Continue buttons.
+             This field is constant.</summary>

```

Comment:
```suggestion
        <summary>Specifies that the message box contains Cancel, Try Again, and Continue buttons.</summary>

```

Link: https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.forms.messageboxbuttons?view=net-6.0&branch=pr-en-us-6940
![image](https://user-images.githubusercontent.com/4403806/127411741-a97260bd-2f06-47af-8bd4-3eaaba1dff9f.png)

Reply:
Tool exclude existing documented APIs but to make this reliable/consistent, I think we should do these changes in the source code in winforms repo and rerun this tool.